mods/wool/init.lua

```diff
@@ -9,7 +9,7 @@ for i = 1, #dyes do
 	local name, desc = unpack(dyes[i])
 
 	minetest.register_node("wool:" .. name, {
-		description = S("@1 Wool", desc),
+		description = S(desc .. " Wool"),
```

Comment:
Dynamic name again. Consider inserting dummy gettext lines like builtin; to ensure it's parsed.

Reply:
I'm aware, can you link me to a source file where builtin does this?

Reply:
Just a note: I've tested this way before and it doesn't translates the string.
This should be reverted.

Reply:
works for me:
![png](https://user-images.githubusercontent.com/1042418/64691885-0c698100-d494-11e9-9162-c752a80678ae.png)

Reply:
Nice! 👍